cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx

```diff
@@ -32,7 +32,11 @@ function mapDispatchToProps(dispatch: any): DispatchToProps {
             dispatch(createAnnotationsAsync(sessionInstance, frame, states));
         },
         onRememberObject(labelID: number): void {
-            dispatch(rememberObject({ activeObjectType: ObjectType.TAG, activeLabelID: labelID }));
+            dispatch(rememberObject({
+                activeObjectType: ObjectType.TAG,
+                activeLabelID: labelID,
+                activeShapeType: undefined,
```

Comment:
Hm, if we do not pass activeShapeType it anyway has ``undefined`` value, right? So, what has changed?

Reply:
It seems that specifying `undefined` changes overwrite behavior here - https://github.com/openvinotoolkit/cvat/blob/develop/cvat-ui/src/reducers/annotation-reducer.ts#L499

If `undefined` is default(aka property is not specified), old activeShapeType from state.drawing remains the same, and if it's specified, activeShapeType becomes overwritten. 
(I've tried to change changing spreading of payload and old state but had no luck)

Reply:
Ok, interesting spread effect

Reply:
But one more question here, ``activeControl `` variable is computed before this spread operator, basing only on payload. So, still not clear.

Reply:
and it sets up as ``tag``.  Okay, I will debug it a bit